core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -299,6 +301,35 @@ class PageModel extends Model
 	 */
 	protected $blnDetailsLoaded = false;
 
+	public function __set($strKey, $varValue)
+	{
+		// Deprecate setting dynamic page attributes if they are set on the global $objPage
+		if (\in_array($strKey, array('pageTitle', 'description', 'robots'), true) && ($GLOBALS['objPage'] ?? null === $this))
```

Comment:
```suggestion
		if (\in_array($strKey, array('pageTitle', 'description', 'robots'), true) && (($GLOBALS['objPage'] ?? null) === $this))
```
(because `===` binds tighter than `??`)

Reply:
Hmm, we can omit the outer brackets then though?

Reply:
Fixed in 58f3728aa3e6ff1603dc7bfd0e80ec96df4993db.